spec/controllers/events_controller_spec.rb

```diff
@@ -3,13 +3,14 @@
 describe EventsController do
   let!(:test_event) { FactoryGirl.create(:event, id: 1) }
   let!(:fut_event) { FactoryGirl.create(:future_event, id: 2) }
```

Comment:
try to only put let statements in blocks they're necessary in. check out my branch for some ideas.

Reply:
this will prevent you from having to change the expect statement for index as well.